### PR TITLE
DRILL-5859: Fixed queued time calculation

### PR DESCRIPTION
**Pull request by prasadns14**
@paul-rogers please review

### CODE REVIEW

**Comment by paul-rogers** [comment]
Thanks for the first-time commit! We like to have the JIRA number in the title; please go ahead and ecit the title so it looks like this:

DRILL-1234: Fixed queued time calculation

Use the actual Drill JIRA ticket number in place of "1234".

File: exec/java-exec/src/main/java/org/apache/drill/exec/work/foreman/rm/EmbeddedQueryQueue.java [hunk 1]
```diff
@@ -93,7 +93,7 @@ public void release() {
     public String queueName() { return "local-queue"; }
   }
 
-  private final int queueTimeoutMs;
+  private final long queueTimeoutMs;
```
Comment (paul-rogers) [inline exec/java-exec/src/main/java/org/apache/drill/exec/work/foreman/rm/EmbeddedQueryQueue.java]: It is not necessary for the timeout to be a long. Timeouts should be on the order of minutes, not decades, so an int is perfectly fine and represents the semantics just a bit better.

File: exec/java-exec/src/main/java/org/apache/drill/exec/work/foreman/rm/QueryQueue.java [hunk 2]
```diff
@@ -61,9 +61,9 @@
 
     private final QueryId queryId;
     private final String queueName;
-    private final int timeoutMs;
+    private final long timeoutMs;
```
Comment (paul-rogers) [inline exec/java-exec/src/main/java/org/apache/drill/exec/work/foreman/rm/QueryQueue.java]: Same comment as above.

File: exec/java-exec/src/main/java/org/apache/drill/exec/work/foreman/rm/DistributedQueryQueue.java [hunk 3]
```diff
@@ -273,9 +273,8 @@ public QueueLease enqueue(QueryId queryId, double cost) throws QueryQueueExcepti
     }
 
     if (lease == null) {
-      int timeoutSecs = (int) Math.round(configSet.queueTimeout/1000.0);
-      logger.warn("Queue timeout: {} after {} seconds.", queueName, timeoutSecs);
-      throw new QueueTimeoutException(queryId, queueName, timeoutSecs);
+      logger.warn("Queue timeout: {} after {} ms.", queueName, configSet.queueTimeout);
```
Comment (paul-rogers) [inline exec/java-exec/src/main/java/org/apache/drill/exec/work/foreman/rm/DistributedQueryQueue.java]: This was actually not wrong. Instead, it mapped the number from the internal ms units into more user-friendly units of seconds. That is, it is more useful to see:

Queue timeout: small after 60 seconds

Than:

Queue timeout: small after 60000 ms.

If we do want to use ms. perhaps format them with commas:

Queue timeout: small after 60,000 ms.

Or show both:

Queue timeout: small after 60,000 ms. (60 seconds)

File: exec/java-exec/src/main/java/org/apache/drill/exec/work/foreman/Foreman.java [hunk 4]
```diff
@@ -455,9 +455,10 @@ private void admit(QueryWorkUnit work) throws ForemanSetupException {
           .build(logger);
     } catch (QueryQueueException e) {
       throw new ForemanSetupException(e.getMessage(), e);
+    } finally {
+      queryManager.markQueueWaitEndTime();
```
Comment (paul-rogers) [inline exec/java-exec/src/main/java/org/apache/drill/exec/work/foreman/Foreman.java]: Good!

File: exec/java-exec/src/main/java/org/apache/drill/exec/work/foreman/rm/EmbeddedQueryQueue.java (hunk 1 above)
Comment (prasadns14) [reply exec/java-exec/src/main/java/org/apache/drill/exec/work/foreman/rm/EmbeddedQueryQueue.java]: Fixed

File: exec/java-exec/src/main/java/org/apache/drill/exec/work/foreman/rm/QueryQueue.java (hunk 2 above)
Comment (prasadns14) [reply exec/java-exec/src/main/java/org/apache/drill/exec/work/foreman/rm/QueryQueue.java]: Fixed

File: exec/java-exec/src/main/java/org/apache/drill/exec/work/foreman/rm/DistributedQueryQueue.java (hunk 3 above)
Comment (prasadns14) [reply exec/java-exec/src/main/java/org/apache/drill/exec/work/foreman/rm/DistributedQueryQueue.java]: I made the change to be consistent with QueryTimeoutException which logs the time in ms.
I made the changes as suggested in both the places.

Queue timeout: small after 60,000 ms. (60 seconds)

**Comment by prasadns14** [comment]
@paul-rogers Made the changes as suggested. please review

**Comment by prasadns14** [comment]
Update the title.

**Pull request closed without merging**